lesson37/src/js/drawer-menu.js

```diff
@@ -0,0 +1,83 @@
+const body = document.body;
+const drawerMenu = document.getElementById("js-drawer-menu");
+const toggleButtonInDrawerMenu = document.getElementById("js-drawer-open-button");
+const drawerMenuNavigation = [...document.querySelectorAll(".js-drawer-menu-nav-link")];
```

Comment:
nits
```suggestion
const drawerMenuNavigations = [...document.querySelectorAll(".js-drawer-menu-nav-link")];
```

Reply:
Thanks for the review.
Fixed!
[c7ad57a](https://github.com/sae-github/JS_lesson/pull/48/commits/c7ad57a6ef7b6ffd64eb6ea5de3512896c49ead3)